src/lib/prefixHex.ts

```diff
@@ -0,0 +1,6 @@
+import { isHex } from './decodeHex';
+
+export function prefixHexIf0xMissing(data: string): string {
+  if (!isHex(data)) return '0x' + data;
+  else return data;
+}
```

Comment:
Just use `addHexPrefix` in `ethereumjs-util`

Reply:
https://github.com/ethereumjs/ethereumjs-util/blob/master/docs/modules/_bytes_.md#const-addhexprefix

Reply:
You also have `stripHexPrefix` and `padToEven` there